startup_scripts/460_contact_groups.py

```diff
@@ -0,0 +1,35 @@
+import sys
+
+from startup_script_utils import (
+    load_yaml,
+    pop_custom_fields,
+    set_custom_fields_values,
+    split_params,
+)
+from tenancy.models import ContactGroup
+
+contact_groups = load_yaml("/opt/netbox/initializers/contact_groups.yml")
+
+if contact_groups is None:
+    sys.exit()
+
+match_params = ["name", "slug"]
```

Comment:
This is not technically needed as this is the default value for _match_params_

Reply:
I removed the line